server/services/index.ts

```diff
@@ -22,6 +22,7 @@ export const services = () => {
   const prisonService = new PrisonService(hmppsAuthClient, welcomeClientBuilder)
 
   return {
+    hmppsAuthClient,
```

Comment:
sharing `hmppsAuthClient` across both sets of services - we only have one token store/redis client